ansible_collections/arista/avd/molecule/eos_designs_deprecated_vars/intended/configs/host1.cfg

```diff
@@ -62,11 +62,13 @@ interface Loopback0
    description CUSTOM_ROUTER_ID
    no shutdown
    ip address 192.168.255.101/32
+   ip ospf area 0.0.0.0
```

Comment:
no ospf auth here...

Reply:
yes we never do on loopbacks. 
I would need to add a connected host to trigger it for underlay, which I can do